packages/protocol/contracts/governance/BondedDeposits.sol

```diff
@@ -57,7 +59,7 @@ contract BondedDeposits is IBondedDeposits, ReentrancyGuard, Initializable, Usin
   // Maps voting, rewards, and validating delegates to the account that delegated these rights.
   mapping(address => address) public delegations;
   // Maps a block number to the cumulative reward for an account with weight 1 since genesis.
-  mapping(uint256 => FractionUtil.Fraction) public cumulativeRewardWeights;
+  mapping(uint256 => int256) public cumulativeRewardWeights;
```

Comment:
I believe it's dangerous not to encapsulate the `int256` into a Fixidity type. If anybody by mistakes uses as a `int256` it might break things.

Reply:
and totally not evident for an unaware user that `int256` is actually a fixed-point number